src/share/util/shr_dmodel_mod.F90

```diff
@@ -771,7 +769,7 @@ subroutine shr_dmodel_readstrm(stream, pio_subsystem, pio_iotype, pio_iodesc, gs
 
   call t_stopf(trim(lstr)//'_setup')
 
-  if (pio_iotype == iotype_std_netcdf) then
+  if (pio_iotype == pio_iotype_netcdf) then
```

Comment:
Ok, that is fine.

In the future, do you still plan on supporting the case when PIO_TYPENAME is not explicitly set?